internal/extsvc/azuredevops/events.go

```diff
@@ -3,20 +3,27 @@ package azuredevops
 import (
 	"encoding/json"
 	"strconv"
+	"strings"
 	"time"
 )
 
 var (
-	PullRequestCreatedEventType AzureDevOpsEvent = "git.pullrequest.created"
-	PullRequestMergedEventType  AzureDevOpsEvent = "git.pullrequest.merged"
-	PullRequestUpdatedEventType AzureDevOpsEvent = "git.pullrequest.updated"
+	PULL_REQUEST_APPROVED_TEXT                  = "approved pull request"
+	PULL_REQUEST_APPROVED_WITH_SUGGESTIONS_TEXT = "has approved and left suggestions in pull request"
+	PULL_REQUEST_REJECTED_TEXT                  = "rejected pull request"
+	PULL_REQUEST_WAITING_FOR_AUTHOR_TEXT        = "is waiting for the author in pull request"
```

Comment:
camel/PascalCase for the win (https://sourcegraph.slack.com/archives/C3B3SDBMY/p1677768847277869)
:)

(And also uniform with vars below)
```suggestion
	PullRequestApprovedText                  = "approved pull request"
	PullRequestApprovedWithSuggestionsText = "has approved and left suggestions in pull request"
	PullRequestRejectedText                  = "rejected pull request"
	PullRequestWaitingForAuthorText        = "is waiting for the author in pull request"
```